editor/editor_settings.cpp

```diff
@@ -1497,10 +1497,35 @@ void EditorSettings::set_favorites(const Vector<String> &p_favorites) {
 	}
 }
 
+void EditorSettings::set_favorite_properties(const HashMap<StringName, PackedStringArray> p_favorite_properties) {
+	favorite_properties = p_favorite_properties;
+	String favorite_properties_file;
+	if (Engine::get_singleton()->is_project_manager_hint()) {
+		favorite_properties_file = EditorPaths::get_singleton()->get_config_dir().path_join("favorite_properties");
```

Comment:
When is this relevant for Project Manager?

Reply:
Copy-and-paste mistake, fixed.